client/rest-high-level/src/test/java/org/elasticsearch/client/security/ClearRealmCacheResponseTests.java

```diff
@@ -54,7 +53,8 @@ public void testParseFromXContent() throws IOException {
             XContentParser parser = JsonXContent.jsonXContent.createParser(
                 NamedXContentRegistry.EMPTY,
                 DeprecationHandler.THROW_UNSUPPORTED_OPERATION,
-                "{" + nodesHeader + clusterName + "\"nodes\" : { \"id1\": { \"name\": \"a\"}, \"id2\": { \"name\": \"b\"}}}"
+                "{" + nodesHeader + clusterName + """
+                    "nodes" : { "id1": { "name": "a"}, "id2": { "name": "b"}}}"""
```

Comment:
If we took out the trailing comma from `nodesHeader` and `clusterName`, then we could do something like:
```suggestion
                """
                    {
                      %s,
                      %s,
                      "nodes" : { "id1": { "name": "a"}, "id2": { "name": "b"}}
                    }
                    """.formatted(nodesHeader, clusterName)
```